docs/gates.md

```diff
@@ -1,46 +1,224 @@
+various simulators. See [noise documentation](noise.md) for more details.
 
+Many arithmetic operators will also work in the expected way when applied to
+gates.  For instance, ``cirq.X ** 0.5`` represents a square root of X gate.
+These can also be applied to Operators for a more compact representation, such
+as ``cirq.X(q1) ** 0.5`` will be a square root of X gate applied to the q1
+qubit.  This functionality depends on the "magic methods" of the gate
+being defined (see below for details).
 
-### Magic Methods
+Gates can be converted to a controlled version by using `Gate.controlled()`.
+For instance, `cirq.X.controlled()` returns a `cirq.CNOT` gate.
+Operations have similar functionality `Operation.controlled_by()`, such as
+`cirq.X(q0).controlled_by(q1)`.
+
+
+## Common gates
+
+Cirq supports a number of gates natively, with the opportunity to extend these
+gates for more advanced use cases.
+
+### Measurement gate
+
+**cirq.MeasurementGate** This is a measurement in the computational basis.
+This gate can be applied to a variable number of qubits.  The function
+``cirq.measure(q0,q1,...)`` can also be used as a short-hand to create a
+``MeasurementGate`` .
+
+
+### Single qubit gates
+
+Most single-qubit gates can be thought of as rotation around an axis in the
+[Bloch Sphere](https://en.wikipedia.org/wiki/Bloch_sphere) representation and
+are usually referred to by their axis of rotation.
+
+
+**cirq.Z / cirq.ZPowGate / cirq.rz** Rotations about the Pauli ``Z`` axis.
+The matrix of `cirq.Z**t` and the equivalent representation
+`cirq.ZPowGate(exponent=t)` is ``exp(i pi |1><1| t)`` whereas the matrix of
+`cirq.rz(θ)` is `exp(-i Z θ/2)`.  Since computation is often done in the
+Z-basis, these are implemented as phase changes on later operations on
+[Google devices](google/devices.md) instead of a physical modification applied
+to the qubits.
+
+**cirq.Y / cirq.YPowGate / cirq.ry** Rotations about the Pauli ``Y`` axis.
+The gate `cirq.Y**t` and the equivalent representation
+`cirq.YPowGate(exponent=t)` are measured in terms of 180 degree turns, while
+the function `cirq.ry(θ)` uses a radian angle of rotation.
+
+**cirq.X / cirq.XPowGate / cirq.rx** Rotations about the Pauli ``X`` axis.
+The gate `cirq.X**t` and the equivalent representation
+`cirq.XPowGate(exponent=t)` are measured in terms of 180 degree turns, while
+the function `cirq.rx(θ)` uses a radian angle of rotation.
+
+**cirq.PhasedXPowGate**
+This gate is a rotation about an axis in the XY plane of the Bloch sphere.
+The ``PhasedXPowGate`` takes two parameters, ``exponent`` and ``phase_exponent``.
+The gate is equivalent to the circuit `───Z^-p───X^t───Z^p───` where `p` is the
+`phase_exponent` and `t` is the `exponent`.
+
+**cirq.H / cirq.HPowGate** The Hadamard gate is a rotation around the X+Z axis.
+``cirq.HPowGate(exponent=t)`` is a variable rotation of t turns around this
+axis. ```cirq.H` is a π rotation and is equivalent to
```

Comment:
It looks like the backticks around `cirq.H` are incorrect (three on the left and one on the right, should be two on both sides).

Reply:
Fixed.